### PR TITLE
Create new research issue template

### LINKED ISSUE #267: 🚧Create new research issue template
### I have done the items below . . .

- [X] I have updated the title by replacing the '**_\<title\>_**' section.

### Description

Create a custom issue template that is for the purpose of doing research.

### Acceptance Criteria

**This issue is finished when:**
- [x] Research label auto added when issue is created
- [x] Text area for describing what to research
- [x] Text area to put research results
- [x] Set the checkbox at the very top the value below:
``` diff
- I have updated the title by replacing the <title> section
+ I have updated the title and left the 🔬 emoji behind.
```

- [x] The emoji used in the title should be 🔬
- [x] Set the text under the **Description** section label to the result below
``` diff
- A description of feature/change in the user's perspective
+ A description of the research to perform.
```
- [x] Checkbox in acceptance criteria that states:
  - Research complete and issues created if required
- [x] Do not need to have a checkbox in the **ToDo Items** section that asks that a draft pull request is created
  - This is because research items are not meant to make ANY changes to ANYTHING in the repo at all.  They are for research only

### ToDo Items

- [x] Draft pull request created and linked to this issue
- [X] Priority label added to issue (**_low priority_**, **_medium priority_**, or **_high priority_**)
- [X] Issue linked to the proper project
- [X] Issue linked to proper milestone

### Issue Dependencies

_No response_

### Related Work

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

💡For more information on branching, refer to the projects [branching documentation](../../Documentation/Branching.md).

- [x] Pull request title matches the title of the associated issue.

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out and let's discuss it.

- [x] A preview-label label has been added to the pull request.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - This does not mean that the tests pass in both the local and CI environment.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/research-issue-template.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
+name: 🔬 Research Item
+description: Create an issue to perform some research.
```
Comment (kselena) [inline .github/ISSUE_TEMPLATE/research-issue-template.yml:2]: Remove "some"
```suggestion
description: Create an issue to perform research.
```

File: .github/ISSUE_TEMPLATE/research-issue-template.yml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+- type: textarea
+  attributes:
+    label: Acceptance Criteria
+    description: |
+      The list of items that need to be completed to satisfy the definition of done.
+    value: |
+      **This issue is finished when:**
+      - [ ] https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%94%AC%20research label added to this issue.
+      - [ ] Research complete and issues created if required
```
Comment (kselena) [inline .github/ISSUE_TEMPLATE/research-issue-template.yml:39]: Reword
```suggestion
      - [ ] Research complete and issues created (if needed)
```

File: .github/ISSUE_TEMPLATE/research-issue-template.yml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  attributes:
+    label: Issue Dependencies
+    placeholder: |
+      No Issue Dependencies Added.
+      Example:
+        - #10 Create new class
+        - #20 Add unit tests to service
+        - #30 Cleanup code in UI
+    description: |
+      A list of issues that are dependencies of this issue and need to be finished first.
```
Comment (kselena) [inline .github/ISSUE_TEMPLATE/research-issue-template.yml:64]: Reword
```suggestion
      A list of dependencies that need to be finished first.
```

**Review by kselena — CHANGES_REQUESTED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/research-issue-template.yml [hunk 4]
```diff
@@ -36,7 +36,7 @@ body:
     value: |
       **This issue is finished when:**
       - [ ] https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%94%AC%20research label added to this issue.
-      - [ ] Research complete and issues created if required
+      - [ ] Research complete and issues created (if needed).
       - [ ] If any issues were created, issues are added to the _**Related Work**_ section below.
```
Comment (kselena) [inline .github/ISSUE_TEMPLATE/research-issue-template.yml:40]: Reword
```suggestion
      - [ ] If any issues were created, they have been added to the _**Related Work**_ section below.
```

**Review by kselena — CHANGES_REQUESTED** [review]

**Review by kselena — APPROVED** [review]

**Pull request merged**